### ISSUE TITLE
jTable with simple ASP

### ISSUE THREAD

**Issue by emeggiolaro** [volosoft/jtable#2168]
Hello, can I use jTable with ASP (not .NET ASPX)?
Thanks

**Comment by falwickster** [comment]
@emeggiolaro Yes you can use jTable on ASP.NET MVC but don't include aspnetpagemethods.js file to your project, this script file is used only for Web Forms
